backend/database/hwSetDB.py

```diff
@@ -1,10 +1,11 @@
-import database.database_methods as db
+import database as db
+from models.HWSet import HWSet
 
 def queryAvailability(projectName, HWSetNum):
    try:
       client = db.get_database()
-      db = client.SWELabProjectHardwareSets
-      collection = db[projectName]
+      projDb = client.SWELabProjectHardwareSets
+      collection = projDb[projectName]
```

Comment:
Why are we querying by projectName? Is this assuming that the hardware set availability is different depending on which project it is?